resources/default_pipeline_environment.yml

```diff
@@ -201,9 +201,6 @@ steps:
       - 'tests'
     testReportFilePath: 'cst-report.json'
   cloudFoundryCreateService:
-    cloudFoundry:
-      apiEndpoint: 'https://api.cf.eu10.hana.ondemand.com'
-      serviceManifest: 'service-manifest.yml'
     dockerImage: 'ppiper/cf-cli'
     dockerWorkspace: '/home/piper'
```

Comment:
Is this section still required in the default_pipeline_environment.yml?

Reply:
No, I switched it to `resources/metadata/cloudFoundryCreateService.yaml` default definition for the respective params.

Reply:
Then please remove these values.
As discussed with Christopher, the default_pipeline_environment does not have any effect on steps implemented in golang. 
-> The default value for "stashContent" can be set as a "resource" in the yaml definition of the step. See: https://github.com/SAP/jenkins-library/blob/5e67c9c01468a217bb2c167be609e06d431145c6/resources/metadata/fortify.yaml#L24

Reply:
Ok, this has also been switched. Good hint